allure-junit5/src/test/java/io/qameta/allure/examples/junit5/AllureStepTest.java

```diff
@@ -0,0 +1,35 @@
+package io.qameta.allure.examples.junit5;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Step;
+import org.junit.jupiter.api.Test;
+
+public class AllureStepTest {
+
+    private static final String GLOBAL_PARAMETER = "global value";
+
+    @Test
+    public void annotatedStepTest() {
+        annotatedStep("local value");
+    }
+
+    @Test
+    public void lambdaStepTest() {
+        final String localParameter = "parameter value";
+        Allure.step(String.format("Parent lambda step with parameter [%s]", localParameter), (step) -> {
```

Comment:
at least we have to show that we can pass lambda without parameters, i.e. `() -> { ... }`
hence – adding one more test-snippet

Reply:
or, just extend this test-method with one more lambda-step without parameter... 

breaking it down would be following more atomic/granular/"unit tests" style approach, that has its cons and pros of course...